shell/store/index.js

```diff
@@ -615,15 +615,17 @@ export const mutations = {
     state.isRancherInHarvester = neu;
   },
 
-  updateNamespaces(state, { filters, all, getters }) {
+  updateNamespaces(state, opt) {
```

Comment:
this should remain as is, getters will just be undefined if not in the object passed to the mutator and it keeps it clear that callers should supply it

to avoid the clash you could do something like 
```
{ filters, all, getters: optGetters }
...
getActiveNamespaces(state, optGetters || getters);
```